src/styles/blocks/main.scss

```diff
@@ -0,0 +1,26 @@
+.main {
+  --column-count: 1;
+  padding: 50px 40px;
+  display: grid;
+  grid-template-columns: repeat(var(--column-count), 200px);
+  gap: 46px 48px;
+  justify-content: center;
+}
+
+@media (min-width: 488px) {
+  .main {
+    --column-count: 2;
+  }
+}
+
+@media (min-width: 768px) {
+  .main {
+    --column-count: 3;
+  }
+}
+
+@media (min-width: 1024px) {
+  .main {
+    --column-count: 4;
+  }
```

Comment:
use scss nesting

Reply:
Still not fixed from the previous review
<img width="846" alt="image" src="https://github.com/mate-academy/layout_catalog/assets/64750958/7f490808-1aa1-41e5-b396-83921291353c">